docs/mkdocs.yml

```diff
@@ -62,18 +62,17 @@ markdown_extensions:
           format: !!python/name:pymdownx.superfences.fence_code_format
 
 nav:
+  - Home:
+      index.md
   - User Guide:
-    - user/index.md
+    - user/what-is-kpops.md
     - Getting Started:
-      - user/getting-started/index.md
       - user/getting-started/setup.md
       - user/getting-started/quick-start.md
       - user/getting-started/teardown.md
     - Examples:
-      - user/examples/index.md
       - user/examples/atm-fraud-pipeline.md
     - References:
-      - user/references/index.md
+      - user/references/components.md
```

Comment:
doesn't this belong to https://github.com/bakdata/kpops/pull/112?

Reply:
It is! This will be merged into @sujuka99 branch. Have a look at the top of the PR. The destination branch is not `main`.